packages/mds-geography/api.ts

```diff
@@ -89,7 +98,17 @@ function api(app: express.Express): express.Express {
           )
         }
 
-        const geographies = summary ? await db.readGeographySummaries(params) : await db.readGeographies(params)
+        const options = { excludeGeographyJSON: summary }
+        if (params.get_unpublished && params.get_published) {
+          throw new BadParamsError('cannot have get_unpublished and get_published both be true')
+        }
+        const geographies = params.get_unpublished
+          ? params.get_published
+            ? [] // should be prevented by the BadParamsError
+            : await GeographyServiceClient.getUnpublishedGeographies(options)
+          : params.get_published
+          ? await GeographyServiceClient.getPublishedGeographies(options)
+          : await GeographyServiceClient.getGeographies(options)
```

Comment:
This much ternary nesting makes me queasy, I think a self-executing closure with if statements might be a little more readable
```suggestion
        const geographies = (() => {
          // if statements and such
          })()
```